### DISCUSSION TITLE
Can the AiInput files be shared among multiple triggers?

### DISCUSSION THREAD

**Discussion by hiro5id** [neilenns/node-deepstackai-trigger#433] [Q&A] [answered]
What I mean by this question is, imagine you have 2 or more triggers defined in the `triggers.json` file.  But all of them use the same `watchPattern` so they are all looking at the same images.

Would that mess up the script and make it trip over itself?

The reason I ask is because I would like to create different kinds of triggers based on the same incoming images, like one trigger for cars, and that will go to a particular telegram channel, and another trigger for people.. etc..

By the way excellent scripting here and all the hard work in documenting everything. Thanks for doing this.

**Comment by neilenns** [comment, marked as answer]
It works fine, you can have multiple triggers registered with the same watch pattern.
